aggregators/1.0/modules/ROOT/pages/aggregators-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+= Considerations for Using Aggregators in a Cluster
+
+[IMPORTANT]
+Aggregators don’t work with multiple workers and a persistent object store because clustering is not available in CloudHub. For details on how workers can be shared or doubled to scale your application and provide high availability, see the xref:runtime-manager::cloudhub-fabric.adoc[CloudHub Fabric] documentation.
+
+Aggregators Module works in a cluster out-of-the-box environment. However, to prevent an unexpected behavior consider the following:
```

Comment:
The rest of this explanation is confusing; I wonder if we can shorten it to something like....
```suggestion
Although Aggregators Module works in a cluster out-of-the-box environment, it is important for you to configure the frequency at which the primary node checks all the other nodes for new aggregations to schedule, to prevent that frequency from conflicting with the aggregation timeout. For example, ....